spec/unit/htty/cli/commands/path_set_spec.rb

```diff
@@ -67,6 +67,34 @@
         built = klass.build_for('x baz', :session => :another_session)
         built.should == nil
       end
+
+      it 'should correctly URL escape simple paths' do
+        built = klass.build_for('path b@r', :session => :a_session)
+        built.should be_instance_of(klass)
+        built.arguments.should == ['b%40r']
+        built.session.should   == :a_session
+      end
+
+      it 'should correctly URL escape multi-part paths' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.